src/voting-contracts/contracts/Proposal.sol

```diff
@@ -86,7 +92,7 @@ contract Proposal is Pausable {
 
     /// proposal details are editable till start block height is not reached
     modifier isEditable() {
-        if (proposal.startBlock < block.number) {
+        if (proposal.startBlock <= block.number) {
```

Comment:
we will start voting at start block height. So it should not be editable at this height. 
change back to `<` instead of `<=`

Reply:
The check is correct. I have updated the condition for readability.